CHANGELOG.md

```diff
@@ -27,6 +27,9 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - Add BBB presentation upload
```

Comment:
Your commit message is not descriptive enough. There is a typo. Which features are added ? What is missing to finish it ?